pronufa.py

```diff
@@ -0,0 +1,83 @@
+from my_tools import dna_rna_tools as na
+from my_tools import fastq_tool as fq
+from my_tools import protein_tool as pt
+from typing import Tuple, Dict
+import sys

```

Comment:
Порядок импорта библиотек: сначала стандартные. потом внешние, потом свои, каждый блоек разделяется пустой строкой, после импортов две пустые строки (тут это есть), частичные импорты после полных
```suggestion
import sys
from typing import Tuple, Dict

from my_tools import dna_rna_tools as na
from my_tools import fastq_tool as fq
from my_tools import protein_tool as pt



```